test/functional/fixtures/api/es-next/click/test.js

```diff
@@ -123,4 +123,8 @@ describe('[API] t.click()', function () {
                 });
         });
     });
+
```

Comment:
Why one test is in regression section and the other one is in click? That's the logic behind it?